packages/lesswrong/server/emails/renderEmail.tsx

```diff
@@ -199,17 +199,17 @@ export async function generateEmail({user, to, from, subject, bodyComponent, boi
     throw new Error("No source email address configured. Make sure \"defaultEmail\" is set in your settings.json.");
   }
   
-  const sitename = forumTitleSetting.get();
-  if (!sitename) {
-    throw new Error("No site name configured. Make sure \"title\" is set in your settings.json.");
-  }
-  const taggedSubject = `[${sitename}] ${subject}`;
+  // const sitename = forumTitleSetting.get();
```

Comment:
This should probably be an optional flag added to the function, rather than an in-place change.